lib/widgets/context_menu/context_menu.dart

```diff
@@ -0,0 +1,149 @@
+// reference to: https://pub.dev/packages/contextmenu
+import 'package:flutter/material.dart';
+import 'package:after_layout/after_layout.dart';
+import 'context_menu_area.dart';
+
+const double _kMinTileHeight = 24;
+
+/// The actual [ContextMenu] to be displayed
+///
+/// You will most likely use [showContextMenu] to manually display a [ContextMenu].
+///
+/// If you just want to use a normal [ContextMenu], please use [ContextMenuArea].
+
+class ContextMenu extends StatefulWidget {
+  /// The [Offset] from coordinate origin the [ContextMenu] will be displayed at.
+  final Offset position;
+
+  /// The builder for the items to be displayed. [ListTile] is very useful in most cases.
+  final ContextMenuBuilder builder;
+
+  /// The padding value at the top an bottom between the edge of the [ContextMenu] and the first / last item
+  final double verticalPadding;
+
+  /// The width for the [ContextMenu]. 320 by default according to Material Design specs.
+  final double width;
+
+  const ContextMenu({
+    Key? key,
+    required this.position,
+    required this.builder,
+    this.verticalPadding = 8,
+    this.width = 320,
```

Comment:
no hard code, only hardcoded for minWidth

Reply:
<img width="427" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/43041967/153167e6-505f-4779-bb76-b1451e86d923">
Change to 200 now